core/controlplane/config/templates/cluster.yaml

```diff
@@ -239,7 +239,7 @@ worker:
 #      # If omitted, public subnets are created by kube-aws and used for worker nodes
 #      subnets:
 #      - # References subnets defined under the top-level `subnets` key by their names
```

Comment:
@sathiyas My assumption was given this block of yaml:

```yaml
  nodePools:
    - # Name of this node pool. Must be unique among all the node pools in this cluster
      name: nodepool1
#      # If omitted, public subnets are created by kube-aws and used for worker nodes
#      subnets:
#      - # References subnets defined under the top-level `subnets` key by their names
#        name: ManagedPublicSubnet1
```

One would uncomment it like:

```yaml
  nodePools:
    - # Name of this node pool. Must be unique among all the node pools in this cluster
      name: nodepool1
      # If omitted, public subnets are created by kube-aws and used for worker nodes
      subnets:
      - # References subnets defined under the top-level `subnets` key by their names
        name: ManagedPublicSubnet1
```

which is parsed into:

```json
{
  "nodePools": [
    {
      "subnets": [
        {
          "name": "ManagedPublicSubnet1"
        }
      ], 
      "name": "nodepool1"
    }
  ]
}
```